packages/swirl-components/src/components/swirl-table-row-group/swirl-table-row-group.tsx

```diff
@@ -13,13 +13,40 @@ export class SwirlTableRowGroup {
 
   @Prop() label!: string;
   @Prop() tooltip?: string;
+  @Prop() collapsable?: boolean = false;
+
+  @State() isVisible: boolean = true;
+
+  private toggleShowItems = () => {
+    const rowsContainer = this.el.shadowRoot.querySelector(
+      ".rows-container"
+    ) as HTMLElement;
+    if (this.isVisible) {
+      rowsContainer.style.height = "0";
+    } else {
+      rowsContainer.style.height = "auto";
+      const height = rowsContainer.clientHeight + "px";
+      rowsContainer.style.height = "0";
+      setTimeout(() => (rowsContainer.style.height = height), 0);
```

Comment:
Suggestion: There is a neat trick to be able to animate to auto height with CSS only. Maybe this works here.
https://css-tricks.com/css-grid-can-do-auto-height-transitions/

Reply:
Issue: The first collapsing isn't animated.

Reply:
I tried multiple times to use the css as you described (I left my approach in comments) But i could not get it to work. Either no row or part of a row disappeared, but not more. 
I added a lot of animation scheduling and the first collapse should be animated now. :)